src/lib/zf/attr.c

```diff
@@ -131,6 +131,7 @@ int zf_attr_alloc(struct zf_attr** attr_out)
     return -ENOMEM;
   if( zf_attr_set_from_sysctl(attr) )
     return -ENOSYS;
+  zf_log_stderr(); /*log_file is static and we want default behaviour for new set of attributes*/
```

Comment:
I'm mimicking the behavior in 
https://github.com/andresa-xilinx/tcpdirect/blob/345a7b452994c4cbe1cfbba27345f33abb124c18/src/lib/zf/zf.c#L20
Not sure whether we want to change this. I think log_file has a unique initialization procedure as shown in link above.

Reply:
@pemberso-xilinx More precisely, the problem is that as log_file is global static, we need to make sure we close the previous file (in case it exists) before resetting the value to stderr. That is, ZF_ATTR macro in __zf_attr_reset() will only allow assigning the intended (stderr) value and would not be enough.